Add interfaces/webrtc-ice.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/webrtc-ice.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer commented:
Started adding a test, found IDL errors. Filed https://github.com/w3c/webrtc-ice/issues/35

autofoolip committed:
Add interfaces/webrtc-ice.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/webrtc-ice.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
